utils/config.go

```diff
@@ -86,6 +90,10 @@ type AzurePublishRoot struct {
 // Config is configuration for aptly, shared by all modules
 var Config = ConfigStructure{
 	RootDir:                filepath.Join(os.Getenv("HOME"), ".aptly"),
+	UseAuth:                false,
+	LdapServer:             "127.0.0.1:389",
```

Comment:
I don't think we want these settings to be the default ones. By default, I guess we want them emptly, i.e. no ldap configured.

Reply:
cleared them up and redesigned some of it